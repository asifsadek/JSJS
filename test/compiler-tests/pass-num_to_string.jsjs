val x = 5;
val y = 10;

print_string(num_to_string(x + y) ^ " as a string");
