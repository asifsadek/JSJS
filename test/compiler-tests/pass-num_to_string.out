15 as a string
